### DISCUSSION TITLE
Can’t find my lights?

### DISCUSSION THREAD

**Discussion by SvenssonsJesper** [icanos/hassio-plejd#270] [Q&A]
Hi,

I have just recently started using HA on a Raspberry pi 4 with some Z-wave devices and Shelly. All that was easy setup.

Now I have been trying to get my pled system into HA.
I have 1 DIM-01, 2 CTR-01 & 1 LED-10 in my system.

I have set up the mosquitto MQTT broker and Plejd add on. But I can’t get my different Plejd units to appear.

If I look into the log of the add- on Plejd I can see all the units communicating.

I have tried everything I understand but I am clearly missing something.

Any suggestions what I might be missing?

**Comment by LapplandsCohan** [comment]
I am in the same situation. I recently set up Home Assistant and have connected Hue, Tibber, Easee, Mill, Sonos, and UniFi without too much hassle and now I am trying to add Plejd.

I am using EMQX as local MQTT broker on the HA server, and if I connect to it I see the Plejd messages coming in, but no devices are added in Home Assistant.

How do I get the devices from the messages in the local MQTT broker?

**Reply by SvenssonsJesper** [reply]
I have solved the problem for me. The solution that solved it for me was to add the mqtt username and password in the mqtt config file like this:

logins: []
require_certificate: false
certfile: fullchain.pem
keyfile: privkey.pem
customize:
  active: false
  folder: mosquitto
user: mqttbroker
password: Xxxxxxxxx
